server/portal/apps/projects/managers/unit_test.py

```diff
@@ -0,0 +1,182 @@
+"""Test.
+
+.. :module:: portal.apps.projects.unit_test
+   :synopsis: Projects app unit tests.
+"""
+from __future__ import unicode_literals, absolute_import
+import logging
+import os
+from django.conf import settings
+from portal.apps.projects.managers.base import ProjectsManager
+import pytest
+
+LOGGER = logging.getLogger(__name__)
+
+
+@pytest.fixture()
+def agave_client(mocker):
```

Comment:
there is a 'mock_agave_client' in conftest.py that could be used and in the other places (projects/unit_test.py and models/unit_test.py).

Reply:
Thanks for the heads-up, I was wondering if we had "standard" fixtures.